codice/ddf
# [2.13.x] DDF-4123 Replace platform-filter-delegate with pax web AuthenticationService

##### ABBREVIATED REVIEW BETWEEN 2.13.X AND MASTER IS IN EFFECT
Link to master PR: #3783

#### What does this PR do?
Replace PlatformFilterDelegate with pax web AuthenticationService

- This task will move the logic of adding security filters over to the platform-paxweb-jettyconfig 
fragment and use AuthenticatorService provided by pax.web.service to apply the security filters.
#### Who is reviewing it? 
@brjeter 
@paouelle 
@ryeats 
@bakejeyner 
@bdeining 
@Kjames5269 
@KalCramer 

#### Select relevant component teams: 
@codice/security 
#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@stustison
#### How should this be tested?
- After install, verify FilterInjector was successfully injecting filters in the logs
- Verify you can navigate to the AdminUI and change settings/install features
- Verify you can ingest through Intrigue, search for the products, and download resources
#### What are the relevant tickets?
[DDF-4123](https://codice.atlassian.net/browse/DDF-4123)
#### Checklist:

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
